ui/apps/platform/src/utils/textUtils.ts

```diff
@@ -16,7 +16,18 @@ export function pluralizeHas(len) {
     return len === 1 ? 'has' : 'have';
 }
 
+export function dedupeDelimtedString(value: string, delimiter = ','): string[] {
+    return Array.from(
+        new Set(
+            value.split(delimiter).map(
+                (v) => (typeof v === 'string' ? v.trim() : v) // type assertion to ensure string
```

Comment:
Curious - in what scenarios is it possible for `v` to _not_ be a string? Or is this a case where TypeScript loses type information somehow?

Reply:
Double checked as recently as Friday that `''.split('\n')` returns `['']` so the question might be less whether or not it is a string (it is) but to filter out debris from typos like double delimiter.

Reply:
By the way, **Delimited**

Reply:
When I first wrote this helper function, I got a a TS error, and this type assertion silenced the error. However, now when I remove it, the TS compiler is fine, so I must have adjusted something else about the function, or the call site, that made type inference work.

I'll remove the assertion from the PR.